bin/sync_vsc_users.py

```diff
@@ -90,7 +91,7 @@ def main():
     (last_timestamp, start_time) = retrieve_timestamp_with_default(
         SYNC_TIMESTAMP_FILENAME,
         start_timestamp=opts.options.start_timestamp,
-        default_timestamp="201710230000Z")
+        default_timestamp="201710230000")
```

Comment:
why do you pass this argument at all? If has a default value, why not use that?

Reply:
Because we might not want to go back in time that much?

Reply:
why not? If there is no file or option give, it should go back all the way no? 

On what basis did you pick this timestamp?

Reply:
Hmm, good question. I recall this being a point in time where we have all "correct" information, but we might want to regularly move this up? Actually, I've no good idea.

Reply:
It depends on what you mean with correct :wink: 

If nothing is given, it should start at the beginning of time no?

Reply:
use an invalid timestamp (using none will probably pickup some other one.
default timestamp makes no sense. eitehr there is a timestamp file and it is used. or there is nt, and something is either very wrong or a new installation.

Reply:
There is a default timestamp in vsc-utils anyway. Let's remove it here and you keep either file or a commandline option.